osqp_vendors/CMakeLists.txt

```diff
@@ -0,0 +1,5 @@
+cmake_minimum_required(VERSION 3.9)
```

Comment:
There's no need for this package, please remove `osqp_vendors`. Any downstream packages can depend on either `osqp_vendor` or `osqp_eigen_vendor` if needed

Reply:
removed